--- applications/adjoint_tests/working/psyad/algorithm/linear_physics/adjt_stabilise_bl_u_alg_mod.X90	2025-02-17 13:57:41.000000000 +0000
+++ applications/adjoint_tests/working/build_adjoint_tests/algorithm/linear_physics/adjt_stabilise_bl_u_alg_mod.X90	2025-02-17 14:09:11.000000000 +0000
@@ -10,7 +10,7 @@
     use function_space_collection_mod, only : function_space_collection
     use stabilise_bl_u_kernel_mod, only : stabilise_bl_u_kernel_type
     use adj_stabilise_bl_u_kernel_mod, only : adj_stabilise_bl_u_kernel_type
-    use finite_element_config_mod, only : element_order
+    use finite_element_config_mod, only : element_order_h, element_order_v
     use fs_continuity_mod, only : w2
     use constants_mod, only : i_def, r_def
     use setop_random_kernel_mod, only : setop_random_kernel_type
@@ -41,7 +41,7 @@
     real(kind=r_def) :: MachineTol
     real(kind=r_def) :: relative_diff
 
-    vector_space_w2_ptr => function_space_collection%get_fs(mesh,element_order,w2)
+    vector_space_w2_ptr => function_space_collection%get_fs(mesh,element_order_h,element_order_v,w2)
     call u_stabilised%initialise(vector_space=vector_space_w2_ptr, name='u_stabilised')
     call u_initial%initialise(vector_space=vector_space_w2_ptr, name='u_initial')
     call u_final%initialise(vector_space=vector_space_w2_ptr, name='u_final')
